apis/v1alpha1/gateway_types.go

```diff
@@ -702,12 +698,12 @@ type ListenerStatus struct {
 	// shares the same port value, this message reports the combined
 	// status of all such Listeners.
 	//
-	// +required
+	// +kubebuilder:validation:MaxLength=250
```

Comment:
Why is `Port` a string for status but an int32 for spec?

Reply:
Created a follow up issue for this one: https://github.com/kubernetes-sigs/service-apis/issues/357